File: Python_Assignments/Python_Assignment_1/Assignment1_2.py
#   2. Write a program which contains one function named as ChkNum()
#      which accept one parameter as number. If number is even then 
#      if should display "Even number" otherwise display "Odd number"
#      on console.

def ChkNum(No):
    if(No%2 == 0):
        print("Even Number")
    
    else:
        print("Odd Number")

def main():
    print("Please enter your number = ")
    Number = int(input())
    ChkNum(Number)

if(__name__ == "__main__"):
    main()

#   Output:
#   D:\Python\Marvellous Python\Assignment 1>python Program_02.py
#   Please enter your number =
#   11
#   Odd Number

#   D:\Python\Marvellous Python\Assignment 1>python Program_02.py
#   Please enter your number =
#   8
#   Even Number

File: Typescript_Assignments/Typescript_Assignmnet_1/Assignment1_2.ts
//  2. Write a typescript program which contains one function named as Area.
//     That function shuld calculate area of circle. Accept value of radius
//     from user and return its area. Defalut value of PI should be 3.14 if 
//     if not provided by the caller.

function Area(Rad : number, PI : number = 3.14) : number
{
    return PI * Rad**2;
}

console.log("Start of application\n");
console.log("Area of circle is : " + Area(5));
console.log("\nEnd of application");

/*

    Output:
    D:\Marvellous_Infosystems\Angular_with_MEAN_FullStack\Angular_Assignments\Typescript_Assignmnet_01>node Assignment1_2.js
    Start of application

    Area of circle is : 78.5

    End of application

*/